### ISSUE TITLE
Add "hide from search page" option to study display

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1301]
# TL;DR

*Note - this is a feature GARDEN is considering, but hasn't decided whether to prioritize doing now or later*

Transition the "Discoverable" flag to a 3-option choice:

### Discoverable 

- Relabel in form as "Fully discoverable"; okay to leave value as "Discoverable" in the db
- Status Quo behavior as "Discoverable"
- Explanatory text: "List this study on the 'Studies' page once you start it, and auto-email eligible families an invitation to participate."

### Targeted Recruitment

- Continue sending emails to eligible families as usual
- Continue displaying studies in the lab-specific pages (https://lookit.mit.edu/studies/garden/) when they are set to discoverable
- Do not display studies at https://lookit.mit.edu/studies/, regardless of filter settings
- Explanatory text: "Auto-email eligible families an invitation to participate. Show study on your lab page, but not on the general "Studies" page."

### Not Discoverable

- Status quo behavior as "non-discoverable"
- Explanatory text: "Make the study available for participants at its direct link once you start the study. The link can be distributed by you the researcher, but will not be shown on Studies pages or auto-emailed to eligible families.

# Narrative

As a researcher, I want to continue sending targeted recruitment to families, but NOT have random browsers "wander in" to my studies.  This might be because my search criteria are narrow (most browsing families will not be eligible) or the consequences for people clicking into a study despite redtext when not eligible are high (e.g. my IRB requires paying anyone who even enters the study.)

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
Leave important notes/suggestions here for implementation (important files, suggested approaches, etc.)
